browser-switch/src/test/java/com/braintreepayments/browserswitch/BrowserSwitchClientTest.java

```diff
@@ -347,9 +566,48 @@ public void deliverResult_whenActivityIsNotABrowserSwitchListener_throwsIllegalA
         exceptionRule.expectMessage("Activity must implement BrowserSwitchListener.");
 
         sut = BrowserSwitchClient.newInstance(browserSwitchConfig, activityFinder, persistentStore, returnUrlScheme);
-        sut.deliverResult(activity);
+        sut.deliverResult(plainActivity);
+    }
+
+    //endregion
+
+    //region test convenience deliverResult methods
+
+    @Test
+    public void convenience_deliverResultWithActivityListener_forwardsInvocationToPrimaryDeliverResultMethod() {
```

Comment:
I like this naming. It gets confusing with multiple overrides, so calling out the `forwardsInvocationToPrimaryMethod` is super clear 👍